Makefile

```diff
@@ -145,6 +146,14 @@ lint:
 		-A clippy::upper-case-acronyms \
 		-A clippy::vec-init-then-push
 
+# FIXME: fails if --release is added due to broken HTTP API tests
+nightly_lint:
+	cp .github/custom/clippy.toml .
```

Comment:
we `cp` the `clippy.toml` into place and delete it afterwards because it contains syntax that is invalid for stable Clippy